code/nodes/featureSignificance/featureSignificance.q

```diff
@@ -4,8 +4,21 @@
 
 \d .automl
 
-featureSignificance.node.inputs  :`config`features`target!"!+F"
-featureSignificance.node.outputs :`sigFeats`features!"S+"
+// @kind function
+// @category node
+// @fileoverview Apply feature significance logic to data post feature extraction
+// @param cfg   {dict}          Configuration information assigned by the user and related to the current run
+// @param feats {tab}           The feature data as a table 
+// @param tgt   {(num[];sym[])} Numerical or symbol vector containing the target dataset
```

Comment:
This can only be a numeric list, at this point the data will have been encoded if symbol